mix.exs

```diff
@@ -46,14 +46,16 @@ defmodule Hexpm.MixProject do
       {:hackney, "~> 1.7"},
       {:libcluster, "~> 3.0"},
       {:mox, "~> 0.3.1", only: :test},
+      {:ecto, github: "elixir-ecto/ecto", override: true},
```

Comment:
I think you don't need this dep.

Reply:
I need to overwrite the dependency because phoenix_ecto requires `~> 2.1`

Reply:
Ah, ok. It is temporary then until a new phoenix_ecto version is out.